src/components/PropTable.js

```diff
@@ -82,6 +86,7 @@ export default class PropTable extends React.Component {
               <td>{row.propType}</td>
```

Comment:
@arunoda Now we fully use Docgen in case it is present!, pls check the update! Thanks